test/verify/check-shell-host-switching

```diff
@@ -61,6 +61,9 @@ class HostSwitcherHelpers:
           return true;
         })""", address)
 
+        # HACK: Dropping the machine does not terminate SSH connection; https://github.com/cockpit-project/cockpit/issues/19672
+        self.machine.execute(f"pkill -f [s]sh.*{address}; while pgrep -f [s]sh.*{address}; do sleep 1; done")
```

Comment:
Note: the `pkill`, and with it the test, will fail when we actually fix that bug. That'll be a good reminder to drop the hack then (or restrict it to RHEL 8)